### ISSUE TITLE
[2.2.10] Some questions about Extension clean up

### ISSUE THREAD

**Issue by yuxisun1217** [Azure/WALinuxAgent#699]
Hi there,

1. In exthandlers.py line#565:
```
if existing_state_text is None or \
    existing_state == ExtHandlerState.NotInstalled:
```
Should it be this? Seems existing_state is the file name but not content.
```
if existing_state_text is None or \
    existing_state_text == ExtHandlerState.NotInstalled:
```

2. Could you please help to explain that why there's "if existing_state_text is None"? I'm not sure that if you mean that the file is empty(It seems that even if it's empty the existing_state_text is still not None...)

Thanks!

**Comment by brendandixon** [comment]
@yuxisun1217 Good catch! This is old code we inherited and we're continuing to clean it up. I'll look the method over.

**Comment by brendandixon** [comment]
Addressed by #700
